freqtrade/commands/cli_options.py

```diff
@@ -552,4 +552,9 @@ def __init__(self, *args, **kwargs):
         help='Do not print epoch details header.',
         action='store_true',
     ),
+    "hyperopt_ignore_unparam_space": Arg(
+        "-u", "--ignore-unparameterized-spaces",
```

Comment:
To be honest, i'd use shorthand commands sparingly (only for important options that are needed often, like `--config` or `--strategy`).

For the moment, i consider this as a edge-case option - which i'd not like to "waste" a single letter shortcut for (not many people will probably use this (and your usecase is in a script as well) - so it should be fine to type it out.